### DISCUSSION TITLE
Name collision with SwiftUI

### DISCUSSION THREAD

**Discussion by progalgo** [realm/realm-cocoa#8283] [Ideas]
Using the word "App" in API seems a terrible mistake: https://github.com/realm/realm-swift/blob/8f5f6d0863bad30f76753a32523321a8885e8db9/RealmSwift/App.swift#L169

I wish there was another library that did not use such a generic term in API.

**Comment by leemaguire** [comment]
Hi @progalgo, Semantically `App` suits our API as that is what refers to a MongoDB Device Sync App on the backend. If you are facing errors while using `SwiftUI` and `RealmSwift` you need to add the Framework name as a prefix to the type to avoid collisions e.g. `RealmSwift.App`.

**Reply by progalgo** [reply]
That is a terrible naming

**Reply by progalgo** [reply]
Is there any other library that allows to read and write data from/to Atlas with user authentication, that does not have the word App in its API?
